ros_buildfarm/templates/snippet/builder_publish-over-ssh.xml.em

```diff
@@ -22,6 +22,7 @@
                   <execTimeout>120000</execTimeout>
                   <usePty>false</usePty>
                   <useAgentForwarding>false</useAgentForwarding>
+                  <useSftpForExec>false</useSftpForExec>
```

Comment:
The updated publish-over-ssh plugin includes support for exec via the
sftp protocol which should be disabled for our jobs and is by default.